### PR TITLE
Tests to check if graphs are being created correctly

### PR DESCRIPTION
**Pull request by aaronw22**
## Do the following before requesting a review

- [ ] I have written tests covering functions I have added or changed.
- [ ] I have run tests and they all pass.
- [ ] I have ensured all new functions will show up in the `_pkgdown.yml` file.
- [ ] I have updated `DESCRIPTION` with any new package dependencies.
- [ ] I have updated `NEWS.md` with a brief description of my changes.
- [ ] I have updated the package documentation with `devtools::document()`.
- [ ] I have updated the package website with `pkgdown::build_site_github_pages()`.

### CODE REVIEW

**Comment by aaronw22** [comment]
@JackBuckley Debugging suggests this is another case where `check_for_y_var` thinks we have a cont. y-var when its a factor.

````
# Graph with discrete x and y vars
  data <- data.table::CJ(x = factor(1:10), y = factor(1:10))
  data[, fill := runif(100, 0, 100)]

  p <- ggplot(data, aes(x, y, fill = fill)) +
    geom_tile()

  debugonce(save_e61)
  save_e61("plot-single-disc-x-y.svg", p)
```

Returns `Error: Discrete value supplied to continuous scale`

File: tests/testthat/test-save_e61.R [hunk 1]
```diff
@@ -188,145 +211,264 @@ test_that("Does save_data work", {
...
+
+  p2_t <- p2 +
+    labs_e61(title = "Panel graph title text",
+             subtitle = "Panel graph subtitle text",
+             y = "'000")
+
+  # Graph with cont-y var with large values from -1000 to +1000
+  data <- data.frame(x = factor(1:10), y = runif(10, -1000, 1000))
+
+  p3 <- ggplot(data, aes(x, y)) +
```
Comment (aaronw22) [inline tests/testthat/test-save_e61.R:391]: @JackBuckley not sure why this graph is not generating correctly in the single panel or multi-panel

File: tests/testthat/test-save_e61.R [hunk 2]
```diff
@@ -188,145 +211,264 @@ test_that("Does save_data work", {
...
+  expect_snapshot_file(suppressWarnings(save_e61("plot-single-cont-x-y.svg", p)))
+
+  # Graph with discrete x and y vars
+  data <- data.table::CJ(x = factor(1:10), y = factor(1:10))
+  data[, fill := runif(100, 0, 100)]
+
+  p <- ggplot(data, aes(x, y, fill = fill)) +
+    geom_tile()
+
+  expect_snapshot_file(suppressWarnings(save_e61("plot-single-disc-x-y.svg", p)))
```
Comment (aaronw22) [inline tests/testthat/test-save_e61.R:289]: @JackBuckley Debugging suggests this is another case where `check_for_y_var` thinks we have a cont. y-var when its a factor.

Returns `Error: Discrete value supplied to continuous scale`

File: tests/testthat/test-save_e61.R (hunk 1 above)
Comment (JackBuckley) [reply tests/testthat/test-save_e61.R:386]: Fixed this now

File: tests/testthat/test-save_e61.R (hunk 2 above)
Comment (JackBuckley) [reply tests/testthat/test-save_e61.R]: Also fixed this, although geom_tile etc. will look kinda ungly in theme61 unless we make further edits.

**Pull request merged**